modules/puppetdb_fact/spec/plans/init_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'bolt_spec/plans'
+require 'bolt/target'
+
+describe 'puppetdb_fact' do
+  include BoltSpec::Plans
+
+  let(:facts) {
+    {
+      'foo' => { 'fqdn' => 'foo' },
```

Comment:
One nit: you may as well add another fact to one of these nodes just to make sure it's not just the first fact that gets returned.